ut/alarm_table_defs_test.cpp

```diff
@@ -222,21 +136,22 @@ TEST_F(AlarmTableDefsTest, ValidTableDefLookup)
 
 TEST_F(AlarmTableDefsTest, InvalidTableDefLookup)
 {
-  _defs.initialize();
+  _defs.initialize(NOT_A_REAL_PATH);
 
   AlarmTableDef& _def = _defs.get_definition(0, 0);
 
   EXPECT_FALSE(_def.is_valid());
 }
 
-TEST_F(AlarmTableDefsTest, InvalidStateMapping)
+// This test is temporarily disabled during rework of alarms code
+/*TEST_F(AlarmTableDefsTest, InvalidStateMapping)
```

Comment:
You can call this `InvalidStateMapping_DISABLED` - this disables it, but tells you "You have 1 disabled test" at the end of a run, so you don't forget.